### PR TITLE
LPS-115250 Manually add Account Groups

### PR DESCRIPTION
**Pull request by alee8888**
https://issues.liferay.com/browse/LPS-115250

@drewbrokke @jonmak08

### CODE REVIEW

**Comment by alee8888** [comment]
ci:test:sf

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/context/ViewAccountGroupsManagementToolbarDisplayContext.java [hunk 1]
```diff
@@ -48,6 +55,40 @@ public ViewAccountGroupsManagementToolbarDisplayContext(
...
+			});
+	}
+
+	public List<String> getAvailableActions(
+			AccountGroupDisplay accountGroupDisplay)
+		throws PortalException {
+
+		return ListUtil.fromArray("deleteAccountGroups");
+	}
+
```
Comment (pei-jung) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/context/ViewAccountGroupsManagementToolbarDisplayContext.java:91]: This is not needed unless there's condition for making the delete action available (e.g. permission checks)

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_groups_admin/js/AccountGroupsManagementToolbarDefaultEventHandler.es.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import {DefaultEventHandler} from 'frontend-js-web';
+
+class AccountGroupsManagementToolbarDefaultEventHandler extends DefaultEventHandler {
+	deleteAccountGroups(itemData) {
+		if (
+			confirm(
+				Liferay.Language.get('are-you-sure-you-want-to-delete-this')
+			)
+		) {
+			this._updateAccountGroups(itemData.deleteAccountGroupsURL);
```
Comment (pei-jung) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_groups_admin/js/AccountGroupsManagementToolbarDefaultEventHandler.es.js:24]: Inline this

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_groups_admin/view.jsp [hunk 3]
```diff
@@ -38,6 +38,15 @@ ViewAccountGroupsManagementToolbarDisplayContext viewAccountGroupsManagementTool
...
 			>
+
+				<%
+				Map<String, Object> rowData = HashMapBuilder.<String, Object>put(
+					"actions", StringUtil.merge(viewAccountGroupsManagementToolbarDisplayContext.getAvailableActions(accountGroupDisplay))
+				).build();
+
+				row.setData(rowData);
+				%>
+
```
Comment (pei-jung) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_groups_admin/view.jsp:49]: Not needed for now.

**Comment by alee8888** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
See https://github.com/pei-jung/liferay-portal/pull/649. Thanks.

**Pull request closed without merging**